src/hyp3_isce2/packaging.py

```diff
@@ -397,8 +401,10 @@ def make_parameter_file(
     else:
         ref_tag = reference_scene[-4::]
         sec_tag = secondary_scene[-4::]
-    reference_safe = [file for file in os.listdir('.') if file.endswith(f'{ref_tag}.SAFE')][0]
-    secondary_safe = [file for file in os.listdir('.') if file.endswith(f'{sec_tag}.SAFE')][0]
+        reference_safe = [file for file in os.listdir('.') if file.endswith(f'{ref_tag}.SAFE')][0]
+        secondary_safe = [file for file in os.listdir('.') if file.endswith(f'{sec_tag}.SAFE')][0]
```

Comment:
```suggestion
    if reference_safe is None:
        reference_safe = [file for file in os.listdir('.') if file.endswith(f'{ref_tag}.SAFE')][0]
        secondary_safe = [file for file in os.listdir('.') if file.endswith(f'{sec_tag}.SAFE')][0]
```
I think I'd just let these be optional and look for them if they are `None`. (could check both but checking reference should be sufficient)